css/_components/alert/_alert.css

```diff
@@ -0,0 +1,77 @@
+.alert {
+  align-items: center;
+  background-color: var(--light);
+  border: 1px solid var(--dark);
+  color: var(--dark);
+  display: flex;
+  font-size: var(--fs-xs-1);
+  margin: var(--space-xxxs);
+  padding: var(--pd-sm);
+  width: min(400px, 90%);
+}
+
+.alert__icon--right {
+  cursor: pointer;
+  margin-left: auto;
+}
+
+/* W/ICON */
+.al-wicon .alert__text {
+  margin: 0 var(--space-sm);
+}
+
+/* OUTLINE */
+.al-primary {
+  background-color: white;
```

Comment:
i feel if you rewrite as ".alert-primary" and such , the code readability would improve

Reply:
Thank you for the suggestion, Mohammed. Although it becomes a bit lengthy for other components, so in order to maintain consistency in naming I abbreviated it. It might seem small for the alert but for something like button-solid-primary it looks lengthy hence this kind of naming!